src/scripts/postTransactionBundles.js

```diff
@@ -0,0 +1,79 @@
+const axios = require('axios');
+const path = require('path');
+const fs = require('fs');
+const { createPatientGroupsPerMeasure } = require('../util/groupUtils');
+const mongoUtil = require('../util/mongo');
+
+// for each group, include the patients that are members of the previous group
+// (ex. creates a group of size 10, a group of size 100, ..., a group of size 20,000)
+const GROUP_SIZES = [10, 90, 900, 9000];
+
+/**
+ * Uploads all transaction bundles from the specified directory
+ * into the database. Creates a group resource containing
+ * all the patients (with their updated ids that are created
+ * during the transaction bundle upload process) in the process.
+ * Uploads the created group resource to the database.
+ *
+ * Note: Synthea creates batch bundles for hospital/practitioner info, which
```

Comment:
Ah yeah, so Synthea creates batch bundles for hospital/practitioner info, and then there's an option to generate a Group resource containing the patient references (so not a bundle). Originally I had disabled that but the data Reece generated had the Group resource file included, so I added code to filter it out (since it has outdated references in it). I'll add a separate comment about it